### PR TITLE
Clean up changelog

### PR DESCRIPTION
**Pull request by kwigley**
### Description

trivial change to update and format changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,42 +43,48 @@
...
 - [@kostek-pl](https://github.com/kostek-pl) ([#3236](https://github.com/fishtown-analytics/dbt/pull/3408))
 - [@matt-winkler](https://github.com/matt-winkler) ([#3387](https://github.com/dbt-labs/dbt/pull/3387))
 - [@tconbeer](https://github.com/tconbeer) [#3468](https://github.com/fishtown-analytics/dbt/pull/3468))
 - [@JLDLaughlin](https://github.com/JLDLaughlin) ([#3473](https://github.com/fishtown-analytics/dbt/pull/3473))
 - [@jmriego](https://github.com/jmriego) ([#3526](https://github.com/dbt-labs/dbt/pull/3526))
 - [@danielefrigo](https://github.com/danielefrigo) ([#3547](https://github.com/dbt-labs/dbt/pull/3547))
 
+## dbt 0.20.1 (August 11, 2021)
+
+## dbt 0.20.1rc1 (August 02, 2021)
```
Comment (kwigley) [inline CHANGELOG.md:60]: added missing release dates

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,42 +43,48 @@
...
 
-## dbt 0.20.1 (Release TBD)
 
 ### Features
 - Adds `install-prerelease` parameter to hub packages in `packages.yml`. When set to `True`, allows prerelease packages to be installed. By default, this parameter is False unless explicitly set to True.
 
 ### Fixes
-- Fix config merge behavior with experimental parser ([#3640](https://github.com/dbt-labs/dbt/pull/3640), [#3637](https://github.com/dbt-labs/dbt/pull/3637))
+- Fix config merge behavior with experimental parser [3637](https://github.com/dbt-labs/dbt/pull/3637)
+- Fix exception on yml files with all comments [3568](https://github.com/dbt-labs/dbt/issues/3568)
```
Comment (kwigley) [inline CHANGELOG.md:68]: added

File: CHANGELOG.md [hunk 3]
```diff
@@ -37,42 +43,48 @@
...
-- Use `SchemaParser`'s render context to render test configs in order to support `var()` configured at the project level and passed in from the cli ([#3564](https://github.com/dbt-labs/dbt/issues/3564), [#3646](https://github.com/dbt-labs/dbt/pull/3646))
-- Partial parsing: check unique_ids when recursively removing macros ([#3636](https://github.com/dbt-labs/dbt/issues/3636), [#3659](https://github.com/dbt-labs/dbt/issues/3659))
+- Use `SchemaParser`'s render context to render test configs in order to support `var()` configured at the project level and passed in from the cli ([#3564](https://github.com/dbt-labs/dbt/issues/3564). [#3646](https://github.com/dbt-labs/dbt/pull/3646))
+- Partial parsing: check unique_ids when recursively removing macros ([#3636](https://github.com/dbt-labs/dbt/issues/3636))
+
+### Under the hood
+- Add tracking for determine why `dbt` needs to re-parse entire project when partial parsing is enabled ([#3572](https://github.com/dbt-labs/dbt/issues/3572), [#3652](https://github.com/dbt-labs/dbt/pull/3652))
```
Comment (kwigley) [inline CHANGELOG.md:81]: added

File: CHANGELOG.md [hunk 4]
```diff
@@ -37,44 +43,52 @@
...
 - [@matt-winkler](https://github.com/matt-winkler) ([#3387](https://github.com/dbt-labs/dbt/pull/3387))
 - [@tconbeer](https://github.com/tconbeer) [#3468](https://github.com/fishtown-analytics/dbt/pull/3468))
 - [@JLDLaughlin](https://github.com/JLDLaughlin) ([#3473](https://github.com/fishtown-analytics/dbt/pull/3473))
 - [@jmriego](https://github.com/jmriego) ([#3526](https://github.com/dbt-labs/dbt/pull/3526))
 - [@danielefrigo](https://github.com/danielefrigo) ([#3547](https://github.com/dbt-labs/dbt/pull/3547))
 
+## dbt 0.20.1 (August 11, 2021)
 
-## dbt 0.20.1 (Release TBD)
+## dbt 0.20.1rc1 (August 02, 2021)
```
Comment (kwigley) [inline CHANGELOG.md:60]: update release dates

File: CHANGELOG.md [hunk 5]
```diff
@@ -244,27 +268,45 @@ Contributors:
...
 - [@drkarthi](https://github.com/drkarthi) ([#2426](https://github.com/fishtown-analytics/dbt/issues/2426), [#2998](https://github.com/fishtown-analytics/dbt/pull/2998))
 
-## dbt 0.19.2 (Release TBD)
+## dbt 0.19.2 (June 28, 2021)
+
+### Fixes
+
+- Fix infinite recursion when parsing schema tests due to loops in macro calls ([#3444](https://github.com/fishtown-analytics/dbt/issues/3344), [#3454](https://github.com/fishtown-analytics/dbt/pull/3454))
+
+## dbt 0.19.2rc2 (June 03, 2021)
```
Comment (kwigley) [inline CHANGELOG.md:277]: added as well

File: CHANGELOG.md [hunk 6]
```diff
@@ -244,27 +268,45 @@ Contributors:
...
 ### Fixes
+
 - Fix references to macros with package names when rendering schema tests ([#3324](https://github.com/fishtown-analytics/dbt/issues/3324), [#3345](https://github.com/fishtown-analytics/dbt/pull/3345))
 
+## dbt 0.19.2rc1 (April 28, 2021)
+
+### Fixes
+
+- Ensure that schema test macros are properly processed ([#3229](https://github.com/fishtown-analytics/dbt/issues/3229), [#3272](https://github.com/fishtown-analytics/dbt/pull/3272))
+- Fix regression for default project/database for BigQuery connections ([#3218](https://github.com/fishtown-analytics/dbt/issues/3218), [#3305](https://github.com/fishtown-analytics/dbt/pull/3305))
```
Comment (kwigley) [inline CHANGELOG.md:292]: added as well

**Comment by kwigley** [comment]
My editor used `prettier` to auto format markdown, if we don't want that diff then I can remove that! I commented on the PR with the things I added.

File: CHANGELOG.md (hunk 6 above)
Comment (leahwicz) [reply CHANGELOG.md:292]: Are we missing all of these b/c we made the commit in the release branch and not here?

**Review by leahwicz — APPROVED** [review]

File: CHANGELOG.md (hunk 6 above)
Comment (kwigley) [reply CHANGELOG.md:292]: The fix was made in the wrong order at the time, it was added to 0.19.latest then cherry-picked to develop. I confirmed the fix is on develop (https://github.com/dbt-labs/dbt/commit/5956a64b0109f7b33681d37fa49364e900b730ac)

**Pull request merged**